assets/css/edd-admin.css

```diff
@@ -2350,18 +2350,6 @@ body.download_page_edd-reports {
 #edd-dashboard-widgets-wrap .metabox-holder {
 	padding-top: 0;
 }
-.edd-reports-wrapper .postbox {
-	border: none;
-	box-shadow: none;
-}
-
-.edd-reports-wrapper form .edd-from-to-wrapper,
-.edd-reports-wrapper form .edd-select,
-.edd-reports-wrapper form .edd-label,
-.edd-reports-wrapper form input.button {
-	vertical-align: middle;
-	float: left;
-}
 
 .edd-reports-wrapper form .edd-label {
 	height: 29px;
```

Comment:
I think all of this can be removed, and replaced with a `margin-bottom: 16px;` to get it to match the spacing above.

Reply:
Sorry, I mean all of the rules on the `.edd-reports-wrapper form .edd-label` element (which I suggest elsewhere we maybe rename as well).